rpm/nginx-ldap-auth.spec

```diff
@@ -19,7 +19,7 @@ Reference implementation of method for authenticating users on behalf of
 servers proxied by NGINX or NGINX Plus.
 
 %prep
-%setup -q
+%setup -q -n nginx-ldap-auth-release-%{version}
```

Comment:
renaming directory from 'nginx-ldap-auth-%version' to 'nginx-ldap-auth-release-%version' seems to be useless. Could  you please justify this change?

Reply:
That's what's in the 0.0.4 release tarball.  After downloading the release, into `rpmbuild/SOURCES` and extracting the spec file into `rpmbuild\SPECS`, the build fails with
```
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.QO2R9V
+ umask 022
+ cd /home/afisher/rpmbuild/BUILD
+ cd /home/afisher/rpmbuild/BUILD
+ rm -rf nginx-ldap-auth-0.0.4
+ /usr/bin/gzip -dc /home/afisher/rpmbuild/SOURCES/nginx-ldap-auth-release-0.0.4.tar.gz
+ /usr/bin/tar -xf -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd nginx-ldap-auth-0.0.4
/var/tmp/rpm-tmp.QO2R9V: line 35: cd: nginx-ldap-auth-0.0.4: No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.QO2R9V (%prep)
```

Reply:
OK, got it.
I suppose we should change directory name packed into a release tarball. Mostly because it's somehow wrote to make people guess proper directory name while it should be %name-%version.

Reply:
I changed release naming (and thus - directory naming inside archives). Please, remove this change from your PR.